include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -371,6 +371,17 @@ SECTIONS
 	_app_smem_rom_start = LOADADDR(_APP_SMEM_SECTION_NAME);
 #endif  /* CONFIG_APP_SHARED_MEM */
 
+	/* Non-cached region of RAM */
```

Comment:
Please, add this section after the application memory, and before the shared memory section. The reason has been discussed in #11528: we would like all the static (fixed) memory regions to be in the beginning of SRAM (e.g. APP memory, no-cache, section, execute-from-SRAM section), leaving a continuous memory area on which we can define dynamic regions (e.g. thread stack).

Reply:
Ok. I have just moved it.